website/content/docs/k8s/crds/index.mdx

```diff
@@ -271,25 +248,24 @@ spec:
 
 </CodeBlockConfig>
 
-~> **NOTE:** If two `ServiceIntentions` resources set the same `spec.destination.name`, the
-last one created will not be synced.
+~> **Note:** If two `ServiceIntentions` resources set the same `spec.destination.name`, the
+last one created is not synced.
 
 ### Consul Enterprise <EnterpriseAlert inline />
 
 Consul Enterprise supports multiple configurations for how Kubernetes namespaces are mapped
 to Consul namespaces. The Consul namespace that the custom resource is registered
 into depends on the configuration being used but in general, you should create your
-custom resources in the same Kubernetes namespace as the service they're configuring and
-everything will work as expected.
+custom resources in the same Kubernetes namespace as the service they configure.
 
 The details on each configuration are:
 
-1. **Mirroring** - The Kubernetes namespace will be "mirrored" into Consul, i.e.
-   service `web` in Kubernetes namespace `web-ns` will be registered as service `web`
+1. **Mirroring** - The Kubernetes namespace is "mirrored" into Consul. For example, the
```

Comment:
```suggestion
1. **Mirroring** - The Kubernetes namespace is mirrored into Consul. For example, the
```
Unsure about the quotation marks -- are we quoting K8s docs or calling out terminology?